### PR TITLE
$shalev - exercise 1

### PR DESCRIPTION
**Pull request by shalev1233**

### CODE REVIEW

**Review by ley3103 — COMMENTED** [review]
u r bot

**Review by ley3103 — COMMENTED** [review]
u r not ben and u r younger than 28 , u dont have a friend named eli

File: yaml/shalev/yaml_exercise1_$shalev.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# string
+name: Ben
+# number
+age: 28
```
Comment (etayruben) [inline yaml/shalev/yaml_exercise1_$shalev.yaml]: you are not 28

File: yaml/shalev/yaml_exercise1_$shalev.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# string
+name: Ben
```
Comment (etayruben) [inline yaml/shalev/yaml_exercise1_$shalev.yaml]: your name is not Ben

File: yaml/shalev/yaml_exercise1_$shalev.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+# number
+age: 28
+isPerson: true
+# array
+food: [pizza,beer, schnitzel]
+# array option 2
+friends:
+   - name: Eli
```
Comment (etayruben) [inline yaml/shalev/yaml_exercise1_$shalev.yaml:10]: i dont think you have a friend names Eli, this is Ben's friend

File: yaml/shalev/yaml_exercise1_$shalev.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+     age: 28
+     isPerson: true
+     isOlder: false
+   - name: E-T
```
Comment (etayruben) [inline yaml/shalev/yaml_exercise1_$shalev.yaml:14]: i dont think you have a friend names E-T.. hes from a movie!

**Review by etayruben — CHANGES_REQUESTED** [review]

File: yaml/shalev/yaml_exercise1_$shalev.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+
+  sisters:
+     or:
+       kids: null
+       age: 12
+       isOlder: false
+       dateOfBirth: 27/10/2008
+
+summary: 'this is my lovely summary'
```
Comment (omershilohpsi) [inline yaml/shalev/yaml_exercise1_$shalev.yaml]: missing multiline summary

**Review by omershilohpsi — CHANGES_REQUESTED** [review]

**Review by benedery — APPROVED** [review]

**Pull request merged**